tests/ats/test_grafana.py

```diff
@@ -1,16 +1,69 @@
-from typing import cast
+import logging
+from typing import List
 
 import pykube
 import pytest
 from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.k8s.deployment import wait_for_deployments_to_run
 
+logger = logging.getLogger(__name__)
 
-# when grafana pod is up, it might be still starting and returning 503
-@pytest.mark.flaky(reruns=10, reruns_delay=10)
-@pytest.mark.functional
-def test_grafana_login_page_available(kube_cluster: Cluster) -> None:
-    srv = cast(pykube.Service, pykube.Service.objects(kube_cluster.kube_client).get_or_none(name="grafana"))
-    if srv is None:
-        raise ValueError("'grafana service not found in the 'default' namespace")
-    login_page_res = srv.proxy_http_get("/login")
-    assert login_page_res.ok
+namespace_name = "monitoring"
+deployment_name= "grafana"
+
+timeout: int = 560
+
+@pytest.mark.smoke
+def test_api_working(kube_cluster: Cluster) -> None:
+    """Very minimalistic example of using the [kube_cluster](pytest_helm_charts.fixtures.kube_cluster)
+    fixture to get an instance of [Cluster](pytest_helm_charts.clusters.Cluster) under test
+    and access its [kube_client](pytest_helm_charts.clusters.Cluster.kube_client) property
+    to get access to Kubernetes API of cluster under test.
+    Please refer to [pykube](https://pykube.readthedocs.io/en/latest/api/pykube.html) to get docs
+    for [HTTPClient](https://pykube.readthedocs.io/en/latest/api/pykube.html#pykube.http.HTTPClient).
+    """
+    assert kube_cluster.kube_client is not None
+    assert len(pykube.Node.objects(kube_cluster.kube_client)) >= 1
```

Comment:
Do we need this test?
According to the comments it's just supposed to be an example.

Reply:
Not absolutely necessary but still nice to make sure that the apiserver is reachable before proceeding with any further tests right ?

Reply:
Ok, why not, but then that's what the comment should say.

Reply:
Updated the comment.